.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "third-party/pybind11"]
```

Comment:
did you mean to add pybind11 as a submodule here? In the instructions below you ask users to clone it directly, so this is not needed

Reply:
Ah okay, should we change the instuctions to: `git submodule update --init --recursive` instead as discussed in https://git-scm.com/book/en/v2/Git-Tools-Submodules ? or should we remove the gitmodules thing?